src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
Specify a font fallback system for the 'font-family' property, in case 'Roboto' is not available on the user`s device.